### PR TITLE
Improve the tool-info modules of ABC and AVR

### PR DESCRIPTION
**Pull request by Po-Chun-Chien**
Changes include:

- add `REQUIRED_PATHS` attributes for ABC and AVR
- parse version of ABC
- pass resource limits to AVR's command lines

### CODE REVIEW

File: benchexec/tools/avr.py [hunk 1]
```diff
@@ -15,13 +15,21 @@ class Tool(benchexec.tools.template.BaseTool2):
     URL: https://github.com/aman-goel/avr
     """
 
+    REQUIRED_PATHS = ["build/"]
```
Comment (PhilippWendler) [inline benchexec/tools/avr.py:18]: A directory with this name looks as if it contains build-time stuff, which is typically not required at runtime. Are you sure that this is not the case here?

File: benchexec/tools/avr.py [hunk 2]
```diff
@@ -15,13 +15,21 @@ class Tool(benchexec.tools.template.BaseTool2):
...
+
     def executable(self, tool_locator):
         return tool_locator.find_executable("avr.py")
 
     def name(self):
         return "AVR"
 
     def cmdline(self, executable, options, task, rlimits):
+        from math import ceil
```
Comment (PhilippWendler) [inline benchexec/tools/avr.py:27]: I don't see a reason why this needs to be a local import, so please make it a standard module-level import.

Comment (Po-Chun-Chien) [reply benchexec/tools/avr.py]: Done in a37525b.

File: benchexec/tools/avr.py (hunk 1 above)
Comment (Po-Chun-Chien) [reply benchexec/tools/avr.py:19]: Yes, the build script of AVR by default places the compiled executables under `build/` and the intermediate object code (`*.o`) under `src/`.

**Pull request merged**